File: src/main/java/org/taktik/icure/services/external/rest/v1/dto/RightDto.java
package org.taktik.icure.services.external.rest.v1.dto;

import java.io.Serializable;

public class RightDto implements Serializable{

    private String userId;
    private boolean read;
    private boolean write;
    private boolean administration;

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public boolean isRead() {
        return read;
    }

    public void setRead(boolean read) {
        this.read = read;
    }

    public boolean isWrite() {
        return write;
    }

    public void setWrite(boolean write) {
        this.write = write;
    }

    public boolean isAdministration() {
        return administration;
    }

    public void setAdministration(boolean administration) {
        this.administration = administration;
    }
}

File: src/main/kotlin/org/taktik/icure/services/external/rest/v1/dto/embed/RightDto.kt
package org.taktik.icure.services.external.rest.v1.dto.embed

import com.github.pozo.KotlinBuilder
import java.io.Serializable

@KotlinBuilder
data class RightDto(
        val userId: String? = null,
        val read: Boolean = false,
        val write: Boolean = false,
        val administration: Boolean = false
) : Serializable
